src/icons/Warning.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react'
+import Svg, { Path } from 'react-native-svg'
+import Colors from 'src/styles/colors'
+
+const Warning = ({ color = Colors.warningDark, size = 16 }: { color?: Colors; size?: number }) => (
```

Comment:
Wait, didn't we have a warning icon already?

Reply:
sadly i could not find one, i went through the whole icons folder